vulnerabilities/tests/test_package_managers.py

```diff
@@ -448,6 +453,52 @@ def test_fetch(self):
         assert self.version_api.get("org.apache:kafka") == VersionResponse(valid_versions=expected)
 
 
+class TestGoproxyVersionAPI(TestCase):
+    @classmethod
```

Comment:
I am not a big fan of using setups as they make much harder to read and understand each of the tests .... What do you gain from this when compared to in-lining the setup as needed in each test method?
IMHO self-contained tests are easier to deal with in almost all cases.

Reply:
Well, I just imitated other tests, I thought it was a tradition. I agree with you, setups are only useful when they are shared by multiple tests.

Reply:
Good catch! IMHO they are rarely useful... so let's enter an issue to fix this and avoid using them everywhere unless absolutely necessary.
I entered https://github.com/nexB/vulnerablecode/issues/619